BookPlayer/Player/Views/ArtworkControl.swift

```diff
@@ -145,17 +151,25 @@ class ArtworkControl: UIView, UIGestureRecognizerDelegate {
         self.artworkImage.layer.cornerRadius = 6.0
         self.artworkImage.layer.masksToBounds = true
 
+        let scale = self.isPlaying ? 1.0 : self.artworkScalePaused
+
+        self.playPauseButtonWidth.constant = self.artworkWidth.constant * scale - self.artworkWidth.constant
+        self.playPauseButtonHeight.constant = self.artworkHeight.constant * scale - self.artworkHeight.constant
+
         // Gestures
         self.pan = UIPanGestureRecognizer(target: self, action: #selector(panAction))
         self.pan.delegate = self
         self.pan.maximumNumberOfTouches = 1
         self.pan.cancelsTouchesInView = true
 
         self.addGestureRecognizer(self.pan!)
-    }
 
-    override func layoutIfNeeded() {
+        self.tap = UITapGestureRecognizer(target: self, action: #selector(tapAction))
+        self.tap.delegate = self
+        self.tap.numberOfTapsRequired = 1
+        self.tap.numberOfTouchesRequired = 1
```

Comment:
default value is already `1` https://developer.apple.com/documentation/uikit/uitapgesturerecognizer/1623580-numberoftouchesrequired